packages/cli/generators/controller/index.js

```diff
@@ -98,25 +98,34 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
     ) {
       return;
     }
-    const modelList = await utils.getArtifactList(
-      this.artifactInfo.modelDir,
-      'model',
-    );
-    const repositoryList = await utils.getArtifactList(
-      this.artifactInfo.repositoryDir,
-      'repository',
-      true,
-    );
+
+    let modelList, repositoryList;
+
+    try {
+      modelList = await utils.getArtifactList(
+        this.artifactInfo.modelDir,
+        'model',
+      );
+
+      repositoryList = await utils.getArtifactList(
+        this.artifactInfo.repositoryDir,
+        'repository',
+        true,
+      );
+    } catch (err) {
+      return this.exit(err);
```

Comment:
Having to catch errors and report them via `this.exit` feels ugly to me, but if there isn't any better way, then I can live with it.